articles/140_topic_and_service_name_mapping.md

```diff
@@ -265,7 +335,7 @@ Here are some examples of translations between ROS topic names and the correspon
 | `/_foo/_bar`      | Topic     | `rt___foo___bar`       |
 | `/_foo/_bar/_baz` | Topic     | `rt___foo___bar___baz` |
 
-### ROS Topic and Service Name Length Limit
+#### ROS Topic and Service Name Length Limit
```

Comment:
the length limit is described at the very beginning of the article, where the complete ros topic cannot exceed 255 characters. Am I confusing things here?